ui/ui.go

```diff
@@ -129,7 +129,7 @@ func (h *prdHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	ct := h.content[p]
 	if ct != nil {
 		// if so, render it
-		if err := renderForPrd(w, ct, h.cfg, h.cfgJson, r); err != nil {
+		if err := renderForPrd(w, ct, h.cfg, r); err != nil {
```

Comment:
this diff makes me think that `prdHandler.cfgJson` should be updated, rather than updating these to all use the reference to the Config object. Otherwise, the two would drift from each other.